uiowa/uiowa
# Static Map URL validation updates

Closes #7593 

# How to test

- `ddev blt ds --site=sandbox.uiowa.edu && ddev drush @sandbox.local uli /admin/content/media/`
- Check that you can now filter by Type 'static map image' 
- Copy the link to a location marker from https://maps.uiowa.edu/?id=1890#!ct/51247, or use this one https://maps.uiowa.edu/?id=1890#!ct/51247,57310,57311?m/579297?mc/41.66227490890242,-91.53613999999999?z/17.856476238675306?lvl/0
- Click Add Media->Static map image, paste the location link in the URL field. Add alt text and save. Hooray, the new validation pattern works!
- Now test the old validation pattern still works. You can use the same map link, but this time strip out everything after the ! and before the first m. Leave the m and everything after it until the next question mark. Remove this question mark and everything after it. Or, use this one: https://maps.uiowa.edu/?id=1890#!m/579297.
- Try adding another map but this time remove the ?m/ or !m/. Validation should fail with an error message.
- Bonus points: test the validation also works when adding via the media library from a wysiwyg. 

# Additional testing
- Remove ` static_map_regex: '%[!?]m\/([^?]*)%'` from `sitenow_media_wyiwyg.settings.yml` (found in `config/default`)
- Now try adding a static map, it should not work and an error message should appear.
